pkgs/servers/zigbee2mqtt/update.sh

```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -I nixpkgs=../../.. -i bash -p nodePackages.node2nix curl jq nix-update common-updater-scripts
+#! nix-shell -i bash -p nodePackages.node2nix curl jq nix-update common-updater-scripts
 
 set -euo pipefail
 
+SCRIPT_DIR=$(dirname "$0")
```

Comment:
```suggestion
SCRIPT_DIR="$(dirname "$0")"
```

or it'll break with spaces in the path